block_writer/README.md

```diff
@@ -14,11 +14,11 @@ Run against an existing cockroach node or cluster.
 ```
 # Build cockroach binary from https://github.com/cockroachdb/cockroach
 # Start it in dev mode (listens on localhost:26257)
-./cockroach start --dev
+./cockroach start --insecure
```

Comment:
Perhaps this section should be taken out as there is no difference between Development and Insecure mode now.